src/app/appsettings.json

```diff
@@ -1,4 +1,7 @@
 {
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=(localdb)\\MSSQLLocalDB;Database=PetHome;Trusted_Connection=True;MultipleActiveResultSets=true"
```

Comment:
BAD BAD BAD BAD BAD BAD
(no twinkie)

Reply:
user secrets is a better way
or a app.local.settings file that is not checked in

Reply:
What is the correct way of doing this, how should we approach this, please provide sources of information?